torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8873,6 +8873,9 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                                      'TestBinaryUfuncs',
                                      'test_reference_numerics_extremal_values',
                                      dtypes=(torch.complex64, torch.complex128)),
+                        # 76046
+                        DecorateInfo(unittest.skip("Skipped!"), 'TestCudaFuserOpInfo', 'test_nvfuser_correctness',
```

Comment:
prefer `unittest.expectedFailure`

Reply:
I set the consistently failing tests to `unittest.expectedFailure`, but some of the other tests are failing occasionally depending on the inputs due to not meeting the accuracy bounds (https://github.com/pytorch/pytorch/issues/76046). I left those with `unittest.skip`.

Reply:
The `toleranceOverride` decorator can be applied instead of skipping the tests if you'd prefer a looser accuracy requirement

Reply:
yes, later we're planning on deciding which ops need to be fixed & which can have a looser accuracy requirement as part of #76046.